modules/microvm/virtualization/microvm/guivm.nix

```diff
@@ -90,7 +90,10 @@
           ];
         };
 
-        imports = import ../../module-list.nix;
+        imports = [
+          ../../../common
+          ../../../desktop
```

Comment:
some nits for these, they could be included by module name and not path. but that can be the next round. same for the appvm and guivm